cvat-ui/src/components/actions-menu/dump-submenu.tsx

```diff
@@ -29,12 +32,13 @@ export default function DumpSubmenu(props: Props): JSX.Element {
         menuKey,
         dumpers,
         dumpActivities,
+        versionsAmount,
     } = props;
 
     return (
         <Menu.SubMenu key={menuKey} title='Dump annotations'>
-            {
-                dumpers.map((dumper: string): JSX.Element => {
+            {versionSubmenu(versionsAmount,
```

Comment:
nit: newline after `{`
if you have to split line for parameters, at least put that statement on its own line